models/Category.js

```diff
@@ -0,0 +1,29 @@
+const { Model, DataTypes } = require('sequelize');
+
+const sequelize = require('../config/connection.js');
+
+class Category extends Model {}
+
+Category.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
```

Comment:
Doesn't need to be changed, but allowNull is assumed to be false if the column has primaryKey: true.